ocp_resources/resource.py

```diff
@@ -223,6 +223,15 @@ class Resource(object):
     api_version = None
     singular_name = None
 
+    class Label:
+        NAME = "app.kubernetes.io/name"
```

Comment:
I think what @rnetser wanted is to have
`app.kubernetes.io` under `class ApiGroup`

Reply:
Yes (i.e add "app.kubernetes.io" under  ApiGroup and add Label.NAME)

Reply:
So when used in cnv-tests it should look like:
`    preallocation_requested_annotation = (
        f"{NamespacedResource.ApiGroup.CDI_KUBEVIRT_IO}/{preallocation_requested}"
    )`?